src/core_plugins/console/api_server/es_5_0/cat.js

```diff
@@ -45,12 +60,20 @@ export default function (api) {
     ],
     ['_cat/indices', '_cat/indices/{indices}']);
   addSimpleCat('_cat/master', api);
+  addNodeattrsCat(api);
   addSimpleCat('_cat/nodes', api);
   addSimpleCat('_cat/pending_tasks', api);
+  addSimpleCat('_cat/plugins', api);
   addSimpleCat('_cat/recovery', api);
+  addSimpleCat('_cat/repositories', api);
   addSimpleCat('_cat/thread_pool', api);
   addSimpleCat('_cat/shards', api);
-  addSimpleCat('_cat/plugins', api);
   addSimpleCat('_cat/segments', api);
-  addNodeattrsCat(api);
+  addSimpleCat('_cat/snapshots', api, null, [
+    '_cat/snapshots', '_cat/snapshots/{repository}'
+  ]);
+  addSimpleCat('_cat/tasks', api);
+  addSimpleCat('_cat/templates', api, [
+    '_cat/templates', '_cat/templates/{template}'
```

Comment:
Should this provide autocomplete for the template name or pattern?

Reply:
Yep, did some digging.  I'm going to revert this to use the generic {name} for now and address with https://github.com/elastic/kibana/issues/13052.  The problem with adding all of these is console makes each request separately, once a minute.  Tentative plan to process(minify) and combine on the server, otherwise it's going to end up being a lot of data.